pybleau/app/model/multi_dfs_dataframe_analyzer.py

```diff
@@ -91,9 +94,31 @@ def __init__(self, convert_source_dtypes=False, data_sorted=True,
 
     # Public interface --------------------------------------------------------
 
+    def concat_to_source_df(self, new_df, **kwargs):
+        """ Concatenate potentially unaligned dataframe to the source_df.
+
+        The index of the input DF must be a subset of the source_df's index,
+        and the alignment will be done using it.
+        """
```

Comment:
Please document the expectation that the source_df and new_df do not share any column names.

Reply:
Added. Thanks.